R/stan_module.R

```diff
@@ -117,6 +124,18 @@ setMethod(
     }
 )
 
+#' model_prep. Populates the model section of a StanModule
+#' @param x A StanModule object
+#' @export
+model_prep <- function(x) {
+    if (any(is.na(x@model), nchar(x@model) < 2)) {
+        if (length(x@priors) > 0) x@priors <- as.list(paste(names(x@priors), "~", x@priors))
+        x@model <- h_bracket(x@priors)
```

Comment:
Do you not need to paste the model and the priors together ? If i'm reading this correctly it looks like the priors just overwrites the model ?